### DISCUSSION TITLE
Brownie_fund_me, ValueError: "constructor... length expected 1 but got 0"

### DISCUSSION THREAD

**Discussion by yellowcheese773** [smartcontractkit/full-blockchain-solidity-course-py#170] [Q&A] [answered]
Hi, when I try run `brownie run scripts/deploy.py --network rinkeby`, I get this error: 

```
MacBook-Air-von-Omer:brownie_fund_me omerkinik$ brownie run scripts/deploy.py --network rinkeby
Brownie v1.16.3 - Python development framework for Ethereum

BrownieFundMeProject is the active project.

Running 'scripts/deploy.py::main'...
  File "brownie/_cli/run.py", line 49, in main
    return_value, frame = run(
  File "brownie/project/scripts.py", line 103, in run
    return_value = f_locals[method_name](*args, **kwargs)
  File "./scripts/deploy.py", line 12, in main
    deploy_fund_me()
  File "./scripts/deploy.py", line 7, in deploy_fund_me
    fund_me = FundMe.deploy({"from": account})
  File "brownie/network/contract.py", line 600, in __call__
    return tx["from"].deploy(
  File "brownie/network/account.py", line 507, in deploy
    data = contract.deploy.encode_input(*args)
  File "brownie/network/contract.py", line 629, in encode_input
    data = format_input(self.abi, args)
  File "brownie/convert/normalize.py", line 20, in format_input
    raise type(e)(f"{abi['name']} {e}") from None
ValueError: constructor Sequence has incorrect length, expected 1 but got 0
```

This "expected" thing sounds familiar and I looked that up, but couldn't fix it. 

Here is my brownie-config:

<img width="559" alt="Bildschirmfoto 2021-10-10 um 16 49 49" src="https://user-images.githubusercontent.com/80122924/136700930-01b2dccc-0251-40d4-a2ea-f6e40de91cac.png">

Here is deploy.py:

<img width="559" alt="Bildschirmfoto 2021-10-10 um 16 50 10" src="https://user-images.githubusercontent.com/80122924/136700958-b0e4a0f2-84fe-487e-9c5e-37c30c1613e8.png">

Here is helpful_scripts.py:

<img width="559" alt="Bildschirmfoto 2021-10-10 um 16 50 02" src="https://user-images.githubusercontent.com/80122924/136700986-c8697d92-aa94-4616-b79d-63e4ddb1b2cb.png">

Here is FundMe.sol, the 2 imports flagged as (problem)  red, is maybe this causing the error?

<img width="559" alt="Bildschirmfoto 2021-10-10 um 16 50 28" src="https://user-images.githubusercontent.com/80122924/136701061-7fa01e5b-5b31-4d1c-9d5d-a2d9e47d1627.png">

I would be really thankful for any help.

**Comment by yellowcheese773** [comment, marked as answer]
I found an answer on Stackexchange, somebody had the same issue. 
If anyone want to look it up: https://ethereum.stackexchange.com/questions/109940/brownie-constructor-sequence-has-incorrect-length-expected-1-but-got-0

**Reply by PatrickAlphaC** [reply]
Way to be resourceful here!
